content/joysticks/winwing/winwing-cdu/_index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: WINWING CDU
+weight: 50
+---
+
+WINWING CDU display support is automatic on run, if a supported plane and a CDU are detected. To prevent MobiFlight from taking over the CDU, disable it in `Extras->Settings->Peripherals`.
+
+## Installing Python
+
+Follow the [guide](/guides/installing-python/).
+
+## General remarks
+
+- Python is a mandatory prerequisite.
+- If MobiFlight has detected the CDU on startup, it will show `->MobiFlight<-` on the display.
+- Do not run SimAppPro in parallel. After using SimAppPro, reconnect device to the USB port. MobiFlight currently relies on the CDU default power-on display settings, which might have been changed by a running  SimAppPro instance.
+- Currently, only **default font** is supported.
```

Comment:
```suggestion
- Only **default font** is supported.
```

Reply:
I would also move this to the end of the list, it is the least important of the notes.